WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -125,12 +125,14 @@ - (void)updateBlogSettings:(RemoteBlogSettings *)remoteBlogSettings
                    failure:(void (^)(NSError *error))failure;
 {
     NSParameterAssert([blogID isKindOfClass:[NSNumber class]]);
-
     NSDictionary *parameters = @{ @"blogname" : remoteBlogSettings.name,
                                   @"blogdescription" : remoteBlogSettings.desc,
                                   @"default_category" : remoteBlogSettings.defaultCategory,
                                   @"default_post_format" : remoteBlogSettings.defaultPostFormat,
-                                  @"blog_public" : remoteBlogSettings.privacy
+                                  @"blog_public" : remoteBlogSettings.privacy,
+                                  @"jetpack_relatedposts_enabled" : remoteBlogSettings.relatedPostsEnabled,
```

Comment:
Since these are new settings, and set to Optional in the CoreData model, is it possible that this method could get called, before the `Blog`'s new fields get populated?.

If so, maybe we could just have default values set in `remoteBlogSettingFromBlog` ?